src/EditorFeatures/Core/Implementation/Classification/ClassificationUtilities.cs

```diff
@@ -37,9 +37,10 @@ public static void Convert(ClassificationTypeMap typeMap, ITextSnapshot snapshot
         {
             foreach (var classifiedSpan in list)
             {
+                var classificationType = typeMap.GetClassificationType(classifiedSpan.ClassificationType) ?? typeMap.GetClassificationType(ClassificationTypeNames.Identifier);
```

Comment:
Any reason we can't just explicitly check for the types we know we added but didn't define formation definitions for yet? Might be easier than a fallback that means we'll forget to add a format definition, and things magically kinda work?